setup.py

```diff
@@ -72,13 +72,54 @@ def build_extension(self, ext):
     raise RuntimeError('Unable to determine version.')
 
 
-install_requires = ['attrs>=17.3.0', 'chardet>=2.0,<4.0',
-                    'multidict>=4.0,<5.0',
-                    'async_timeout>=3.0,<4.0',
-                    'yarl>=1.0,<2.0']
-
-if sys.version_info < (3, 7):
-    install_requires.append('idna-ssl>=1.0')
+def get_environment_marker_support_level():
+    """
+    Copied from https://github.com/pytest-dev/pytest/blob/master/setup.py
```

Comment:
If you copy code from third party project, you need to add this reference to NOTICE file as ASF license stands for.

Reply:
I will drop this copied code